subsys/bluetooth/mesh/Kconfig

```
@@ -203,12 +203,15 @@ config BT_MESH_PROXY_USE_DEVICE_NAME
 
 config BT_MESH_PROXY_FILTER_SIZE
 	int "Maximum number of filter entries per Proxy Client"
-	default 3
+	default 16
 	range 1 32767
 	depends on BT_MESH_GATT_PROXY
 	help
 	  This option specifies how many Proxy Filter entries the local
-	  node supports.
+	  node supports. This helps in reducing unwanted traffic getting sent to
+	  the proxy client. This value is application specific and should be large
+	  enough so that proxy client can communicate with several devices through
+	  this proxy server node using the default whitelist filter type.
```

```suggestion
	  the proxy client. This value is application specific and should reflect the
	  the chosen filter type (whitelist/blacklist) and the number of devices the
	  proxy client will communcate with.
```
If you keep it as it is it's no big deal, but I feel the blacklist feature is somewhat neglected in its current state.

---

This is on purpose because the choice of the blacklist is not in the user's hand as a default. According to the spec, the whitelist is a default. I will leave this as is.

---

Fair enough 👍

---

probably my comment is a bit stupid but shouldn't we follow inclusive language in KConfig? I mean `(whitelist/blacklist)`.
There was a huge patch recently to fix documentation according to inclusive language.